tuf/log.py

```diff
@@ -113,6 +113,9 @@
 file_handler = None
 
 # Set the logger and its settings.
+# Note: we're configuring the top-level hierarchy for the tuf package,
+# therefore we explicitly request the 'tuf' logger, rather than following
+# the standard pattern of logging.getLogger(__name__)
```

Comment:
We could avoid this if we move initialising the top-level logger into `tuf/__init__.py`, but as we will still want this module and its API to be able to configure the logger hierarchy it didn't seem particularly beneficial to make that change.